files/amazon-2/sge_preinstall.sh

```diff
@@ -1,5 +1,6 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
why `sh`?

Reply:
This file is mostly the same as the `sge_preinstall.sh` in `files/centos-8`, and we were using `sh` in that script already. Is there a concern against using `sh` as opposed to `bash`?